src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.component.ts

```diff
@@ -489,4 +489,18 @@ export class FileUploadAssessmentComponent implements OnInit, AfterViewInit, OnD
         console.error(error);
         this.jhiAlertService.error(error, null, undefined);
     }
+    allowDrop(ev: any) {
+        ev.preventDefault();
+    }
+    drop(ev: any) {
```

Comment:
You might want to refactor the duplicated code for `allowDrop()` and `drop()` **(2)**.